services/services.go

```diff
@@ -1510,16 +1510,23 @@ func getBurnPageData() (*types.BurnPageData, error) {
 	latestEpoch := LatestEpoch()
 	latestBlock := LatestEth1BlockNumber()
 
+	// Check db to have at least one entry (will error otherwise anyway)
+	burnedFeesCount := 0
+	if err := db.ReaderDb.Get(&burnedFeesCount, "SELECT COUNT(*) FROM chart_series WHERE indicator = 'BURNED_FEES';"); err != nil {
```

Comment:
Remove the `;` at the end of the sql statement, as we never use it, to be consistent